### PR TITLE
libva-utils: update to 2.17.1.

### PR DESCRIPTION
**Pull request by icp1994**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture: **x86_64**

### CODE REVIEW

File: srcpkgs/libva-utils/template [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 configure_args="-Ddrm=true -Dx11=true -Dwayland=true"
 hostmakedepends="pkg-config wayland-devel"
 makedepends="libva-devel libXfixes-devel libXext-devel
  libdrm-devel wayland-devel"
 short_desc="Video Acceleration (VA) API - Utilities"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="icp <pangolin@vivaldi.net>"
 license="MIT"
-homepage="https://01.org/linuxmedia/vaapi"
+homepage="https://github.com/intel/libva-utils"
```
Comment (abenson) [inline srcpkgs/libva-utils/template:13]: More accurate for project homepage? https://www.intel.com/content/www/us/en/developer/topic-technology/open/opensource-media-framework/overview.html

Comment (icp1994) [reply srcpkgs/libva-utils/template:13]: Right, I saw that but the link was so long - felt ugly putting in the template. Just for reference, the previous one is giving "access denied".

Comment (icp1994) [reply srcpkgs/libva-utils/template:13]: https://01.org/linuxmedia this one works - should I use that instead?

Comment (abenson) [reply srcpkgs/libva-utils/template:13]: Sure! That just forwards to the long/actual URL, which is why I think it was (similar to) that before.

**Pull request merged**